### PR TITLE
master install: 234 add requirement check if user is in doil group

### PR DESCRIPTION
**Pull request by daniwe4**
https://github.com/conceptsandtraining/doil/issues/234

### CODE REVIEW

File: setup/install.sh [hunk 1]
```diff
@@ -126,8 +126,8 @@ fi
 # status for check requirements
 doil_status_okay
 
-doil_status_send_message_nl "Set host variable"
-doil_set_host
+doil_status_send_message "Adding group 'doil'"
+doil_system_add_group
 CHECK=$?
```
Comment (tbongers-cat) [inline setup/install.sh:131]: Couldn't you just use `if [[ $? -ne 0 ]]` here (and in the other checks)?

**Review by tbongers-cat — APPROVED** [review]
Looks good to me, just one little remark (see comment) :)

File: setup/install.sh (hunk 1 above)
Comment (daniwe4) [reply setup/install.sh]: done!

**Review by tbongers-cat — APPROVED** [review]
Looks pretty now!

**Pull request merged**